Core/include/Acts/EventData/Measurement.hpp

```diff
@@ -391,4 +399,232 @@ using FittableMeasurement =
 template <typename source_link_t>
 using FittableVolumeMeasurement =
     typename fittable_volume_measurement_helper<source_link_t>::type;
+
+/// A measurement of a fixed-size subspace of the full parameters.
+///
+/// @tparam source_link_t Source link type to connect to the detector readout
+/// @tparam indices_t Parameter index type, determines the full parameter space
+/// @tparam kSize Size of the parameter subset.
+///
+/// The measurement is templated only on the measurement subspace size; its
```

Comment:
A would remove `only` in this context, as it only makes sense if you know the history.